src/graphql/data/util.tsx

```diff
@@ -132,6 +132,15 @@ const URL_CHAIN_PARAM_TO_BACKEND: { [key: string]: InterfaceGqlChain } = {
   base: Chain.Base,
 }
 
+/**
+ * @param chainName parsed in chain name from url query parameter
+ * @returns if chainName is a valid chain name, returns the backend chain name, otherwise returns undefined
+ */
+export function getValidUrlChainName(chainName: string | undefined): Chain | undefined {
+  const validChainName = chainName && URL_CHAIN_PARAM_TO_BACKEND[chainName]
+  return validChainName ? validChainName : undefined
+}
+
```

Comment:
this seems p similar to the one below `validateUrlChainParam`. Could we reuse the existing code?

Reply:
I was originally using the function below, however this has the important distinction that it supports url chain params that are supported by subgraph but not necessarily supported by the GQL BE, whereas the fn below requires it is supported by the GQL BE for Explore and TDP